tests/testthat/test_create_map.R

```diff
@@ -0,0 +1,29 @@
+library(gcammaptools)
```

Comment:
Removed this file because it was outdated.  

See commit:  https://github.com/JGCRI/gcammaptools/pull/51/commits/077d96ec505267be96030fa384540deb3aedc5d3